src/server/server-dev.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+import path from 'path'
+import express from 'express'
+import webpack from 'webpack'
+import webpackDevMiddleware from 'webpack-dev-middleware'
+import webpackHotMiddleware from 'webpack-hot-middleware'
+import config from '../../config/webpack/webpack.dev.config';
+
+const app = express(),
+  DIST_DIR = __dirname,
```

Comment:
it is not a good approach to set constants in this way
it works but I would recommend to go with such way:

```
const a = 
const b = 
const c =   
```

Reply:
done